### PR TITLE
adds urbit (new formula)

### PR DESCRIPTION
**Pull request by joemfb**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

---

formerly headonly, but now has tagged releases. fails to pass `brew audit` due to existing headonly formula (and currently-exceeded github rate-limits).

see Homebrew/homebrew-head-only#205, urbit/urbit#721

### CODE REVIEW

File: Formula/urbit.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  depends_on "libsigsegv"
+  depends_on "openssl"
+
+  depends_on "libtool" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+
+  def install
+    ENV.deparallelize
```
Comment (zmwangx) [inline Formula/urbit.rb]: This could probably be removed? https://github.com/urbit/urbit/pull/332

Comment (joemfb) [inline Formula/urbit.rb]: I was merely aiming for maximum safety -- I'm honestly not sure.

/cc @juped

**Comment by bfontaine** [comment]
It’d be awesome if you could make a PR on the [`homebrew/head-only` tap](https://github.com/Homebrew/homebrew-head-only) to migrate the formula here. All you have to do is to remove the formula and add an entry in [`tap_migrations.json`](https://github.com/Homebrew/homebrew-head-only/blob/master/tap_migrations.json). See https://github.com/Homebrew/homebrew-head-only/pull/215 for an example.

File: Formula/urbit.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    system "make", "BIN=#{bin}", "LIB=#{share}"
+  end
+
+  test do
+    require "open3"
+    Open3.popen3("#{bin}/urbit") do |_, o, e, ps|
```
Comment (MikeMcQuaid) [inline Formula/urbit.rb]: Check and see if `assert_match "simple usage:", shell_output("#{bin}/urbit" 2>&1", 1)` works instead.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by joemfb** [comment]
@bfontaine, is that in addition to, or in place of this PR?

**Comment by bfontaine** [comment]
@joemfb It’s in addition to this one. One PR on the core tap (that one); one PR for the head-only tap.

**Comment by bfontaine** [comment]
Thank you for your contribution to Homebrew, @joemfb; we appreciate it! 🎉

**Pull request closed without merging**